database/sql/src/main/scala/cromwell/database/slick/MetadataSlickDatabase.scala

```diff
@@ -127,14 +127,13 @@ class MetadataSlickDatabase(originalDatabaseConfig: Config)
     runTransaction(action, timeout = timeout)
   }
 
-  override def streamMetadataEntries(workflowExecutionUuid: String,
-                                     fetchSize: Int): DatabasePublisher[MetadataEntry] = {
+  override def streamMetadataEntries(workflowExecutionUuid: String): DatabasePublisher[MetadataEntry] = {
     val action = dataAccess.metadataEntriesForWorkflowExecutionUuid(workflowExecutionUuid)
       .result
       .withStatementParameters(
         rsType = ResultSetType.ForwardOnly,
         rsConcurrency = ResultSetConcurrency.ReadOnly,
-        fetchSize = fetchSize)
+        fetchSize = Integer.MIN_VALUE)
```

Comment:
Presumably this works because your experiment was successful, but a negative value `-2147483648` is a bit surprising. Also, a comment would be helpful.
```suggestion
        fetchSize = 1) // Overriding fetch size to smallest-possible to minimize amount of data in memory
```